lib/src/components/NxTabs/__tests__/NxTabs.test.tsx

```diff
@@ -111,38 +84,47 @@ describe('NxTabs', function () {
       </NxTabs>
     );
 
-    expect(component.find('[role="tab"]').first()).not.toHaveClassName('active');
-    expect(component.find('[role="tab"]').last()).toHaveClassName('active');
-    expect(component.find('[role="tabpanel"]')).toHaveText('Content 1');
+    const tabs = getAllByRole('tab');
+    expect(tabs[1]).toHaveTextContent('Tab 1');
+    expect(tabs[0].classList.contains('active')).toBe(false);
+    expect(tabs[1].classList.contains('active')).toBe(true);
+    expect(getByRole('tabpanel')).toHaveTextContent('Content 1');
   });
 
-  it('calls onTabSelect when a tab is clicked', function () {
+  it('calls onTabSelect when a tab is clicked', async function() {
+    const user = userEvent.setup();
     const onTabSelect = jest.fn();
-    const component = mount(
-      <NxTabs activeTab={0} onTabSelect={onTabSelect}>
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-      </NxTabs>
-    );
 
-    component.find('[role="tab"]').simulate('click');
+    const { getByRole } = quickRender({
+      activeTab: 0,
+      onTabSelect
+    });
+
+    await user.click(getByRole('tab'));
 
     expect(onTabSelect).toHaveBeenCalledWith(0);
```

Comment:
Probably more thorough to have the index of the clicked tab and the already-set `activeTab` be two different values

Reply:
> Probably more thorough to have the index of the clicked tab and the already-set `activeTab` be two different values

Updated here: 1b48c75f47c0586c2c307ad5c17a75c7ffa6e1cc